### PR TITLE
update views for submissions

### PR DESCRIPTION
**Pull request by pretendWhale**
## Motivation and Context

- ensures that all submission views work as intended and all routing is up to date

## Your Changes

**Description**:

**Type of change** (select all that apply):

- [x] Other (please specify): multi-course markus rewrite

## Checklist

- [ ] I have performed a self-review of my own code.
- [x] I have fixed any Hound bot comments.

### CODE REVIEW

File: app/assets/javascripts/Components/submission_table.jsx [hunk 1]
```diff
@@ -205,14 +212,20 @@ class RawSubmissionTable extends React.Component {
     }
```
Comment (mishaschwartz) [inline app/assets/javascripts/Components/submission_table.jsx:212]: There's another one to fix at line 204 (above)

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
I'm still getting a bunch of errors when I try to access the repo browser page. Please have a look and fix those.

File: app/assets/javascripts/Components/submission_table.jsx (hunk 1 above)
Comment (mishaschwartz) [reply app/assets/javascripts/Components/submission_table.jsx:212]: sorry I can't comment on the line directly

**Comment by pretendWhale** [comment]
> I'm still getting a bunch of errors when I try to access the repo browser page. Please have a look and fix those.

Yeah, it turns out if you don't pass a course id prop to nested react components, those components don't know the current course id. Who would have thought. Should be fixed now

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**